### ISSUE TITLE
Webpack plugin should support importing JSON

### ISSUE THREAD

**Issue by FredKSchott** [pikapkg/create-snowpack-app#125]
See this issue: currently it doesn't seem to: https://www.pika.dev/npm/snowpack/discuss/355

**Comment by danprince** [comment]
Just run into this too whilst testing the webpack plugin for #131.

The issue here seems to be that the JSON file isn't really a JSON file at the point where Webpack processes it. Is the fix to just handle `.json` with `babel-loader` instead?

```diff
        module: {
          rules: [
            {
-             test: /\.js$/,
+            test: /\.(js|json)$/,
              use: [
                {
                  loader: "babel-loader",
```

Or should we be pointing webpack back to the original json file on disk?

**Comment by FredKSchott** [comment]
Looks like it was our catch-all for the file-loader that was causing trouble. Fixed in #132
